podman_cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -x
+
+sudo podman stop --all
+sudo podman rm --all --force
+sudo podman image rm --all --force
```

Comment:
Removing everything on the host where dev-scripts have been executed could be pretty strong in case other containers/images have been used

Reply:
Indeed. I do sometimes run other things on my dev host. The cleanup logic already looks for things it knows were started by dev-scripts. Is it missing the names of any pods?

Maybe we want to add the script, but not included it in "make clean" so that it's available but not required?

Reply:
> Maybe we want to add the script, but not included it in "make clean" so that it's available but not required?

Agreed

Reply:
We already remove the ironic pod, what is this removing that the current cleanup didn't remove?

Also I don't think we should remove any images by default - otherwise we'll end up re-pulling the images every single test run.

I do occasionally have to prune old images in my test environment, but IMHO that's preferable to wasting time and bandwidth downloading the same container over and over?

Reply:
If we want to have a podman_cleanup target, I think it would be safer to do `sudo podman image prune --all` in stead of unconditionally removing all-the-images (which as previously noted could easily remove things unrelated to the dev-scripts setup).